src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -102,7 +102,7 @@ private void ProcessToken(JToken token)
                     case JTokenType.Bytes:
                     case JTokenType.TimeSpan:
```

Comment:
right fix was in the earlier PR. In ScriptHostOptionsSetup.cs 

```
private void ConfigureFunctionTimeout(IConfigurationSection jobHostSection, ScriptJobHostOptions options)
        {
            if (options.FunctionTimeout != null)
            {
                ValidateTimeoutValue(options, options.FunctionTimeout);
            }
            else
            {
                TimeSpan functionTimeout = _environment.IsDynamic() ? DefaultFunctionTimeoutDynamic : DefaultFunctionTimeout;
                options.FunctionTimeout = functionTimeout;
            }
        }
```
Options is already binding to functiontimeout as timespan. There is no need to convert it back to string.